### PR TITLE
Chapter1/homework/3

### PR DESCRIPTION
**Pull request by carterwelke**
probably messed something up but maybe it'll work

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -9,6 +9,30 @@ class Program
...
+            Lesson3.Demo();
+        }
+        public static int getInt(string message)
+        {
+            Console.Write(message);
+
+            return Convert.ToInt32(Console.ReadLine());
+        }
+
+        public static string getString(string message)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Rename method 'getString' to match pascal case naming rules, consider using 'GetString'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979&bid=16191144)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -9,6 +9,30 @@ class Program
...
+        {
+            Console.WriteLine(message);
+            return Console.ReadLine();
+        }
+
+        public static float getFloat(string message)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Rename method 'getFloat' to match pascal case naming rules, consider using 'GetFloat'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -9,6 +9,30 @@ class Program
     {
         static void Main(string[] args)
         {
+            Lesson3.Demo();
+        }
+        public static int getInt(string message)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Rename method 'getInt' to match pascal case naming rules, consider using 'GetInt'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 5]
```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979&bid=16191144)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 5 above)
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 3 above)
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979&bid=16191144)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+using System.Text;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979&bid=16191144)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979&bid=16191144)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979&bid=16191144)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979&bid=16191144)

**Comment by Almantask** [comment]
Please remove redundant comments

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 8]
```diff
@@ -0,0 +1,54 @@
+using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Person 1:");
+
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Reading 2 different people should be split by 1 function called twice

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979&bid=16191144)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Person 1:");
+            getPersonData();
+            Console.WriteLine("Person 2:");
+            getPersonData();
+
+        }
+        public static void getPersonData()
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Rename method 'getPersonData' to match pascal case naming rules, consider using 'GetPersonData'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979&bid=16191144)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 8 above)
Comment (carterwelke) [reply Src/BootCamp.Chapter/Lesson3.cs]: changes made

**Pull request closed without merging**